.changeset/green-parrots-brush.md

```diff
@@ -0,0 +1,14 @@
+---
+"@salt-ds/theme": minor
+---
+
+Deprecated 6 alpha foundation tokens.
+
+| Token                  |
```

Comment:
Do we need to suggest the nearest neighbour to choose in their absence?

Reply:
Not sure if it's necessary but if you think we should I'm happy to add it. The other alternative is for consumers to define it themselves